src/components/pages/auth/SignUpForm.jsx

```diff
@@ -188,10 +193,12 @@ const SignUpForm = () => {
             checked={formik.values.accept_policy}
             onChange={formik.handleChange}
           />
-          <label className="cursor-pointer select-none" htmlFor="accept_policy">
+          <label className="cursor-pointer select-none mr-2" htmlFor="accept_policy">
```

Comment:
mr-2? i dont see problems with margins
![image](https://user-images.githubusercontent.com/38906004/104691484-03836200-570f-11eb-9dc9-56ea4327fb20.png)

Reply:
Witout mr-2, there is no space between "I agree with" and "Privacy policy" and looks like "I agree withPrivacy policy"